packages/extract-text2/index.js

```diff
@@ -31,8 +31,8 @@ function extractText (outputFilePattern, fileType) {
             test: context.fileType(fileType),
             exclude: loaderConfig.exclude,
             loader: plugin.extract({
-              fallbackLoader: 'style-loader',
-              loader: nonStyleLoaders
+              fallback: 'style-loader',
+              use: nonStyleLoaders
             }),
```

Comment:
We are currently working on webpack-blocks version 1.0, that will fix this completely.
You can disable the warning with `process.noDeprecations = true`